nodes/src/nodes/router_llm/IInstance.py

```
@@ -0,0 +1,70 @@
+
+import copy
+
+from rocketlib import IInstanceBase, Entry
+from ai.common.schema import Question
+from .IGlobal import IGlobal
+
+
+class IInstance(IInstanceBase):
+    IGlobal: IGlobal
+
+    def open(self, object: Entry):
```

Ruff (0.15.7)

[error] Function argument `object` is shadowing a Python builtin

(A002)